Dockerfile

```diff
@@ -1,4 +1,12 @@
-FROM azul/zulu-openjdk:17
-VOLUME /tmp
-COPY target/*.jar app.jar
-ENTRYPOINT ["java","-jar","/app.jar"]
\ No newline at end of file
+FROM openjdk:17-jdk-slim AS build
```

Comment:
for m1 chipped apple laptops, this image will be problem. please use azul/zulu.

Reply:
Is there a need to run it on Docker locally, or is the Dockerfile only used by the server side?